Cargo.toml

```diff
@@ -1,8 +1,18 @@
 [package]
-authors = ["Crypto Garage"]
+authors = [
+  "Crypto Garage",
+  "Yancy Ribbens <email@yancy.lol>",
+]
 edition = "2018"
+homepage = "https://github.com/p2pderivatives/rust-bitcoin-coin-selection/"
+license = "CC0-1.0"
 name = "rust-bitcoin-coin-selection"
+repository = "https://github.com/p2pderivatives/rust-bitcoin-coin-selection/"
```

Comment:
If this is to move to Rust Bitcoin, then I think the repository will be relocated.  Is there any problem with changing this to `"https://github.com/rust-bitcoin/coin-selection/"` with this PR?

Reply:
Mmmm it makes for a broken link but I updated it anyway.